src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerReviewEvidenceNoteTests.cs

```diff
@@ -0,0 +1,334 @@
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            httpContext.AttachToController(ManageEvidenceController);
+            var model = GetValidModel();
+            AddModelError();
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(OrganisationId, EvidenceNoteId) as ViewResult;
+
+            // assert
+            result.ViewName.Should().Be("ReviewEvidenceNote");
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNoteGet_GivenValidOrganisation_and_GivenValidNoteId_BreadcrumbShouldBeSet()
+        {
+            // arrange 
+            var organisationName = "OrganisationName";
+
+            A.CallTo(() => Cache.FetchOrganisationName(A<Guid>._)).Returns(organisationName);
+
+            // act
+            await ManageEvidenceController.ReviewEvidenceNote(OrganisationId, EvidenceNoteId);
+
+            // assert
+            Breadcrumb.ExternalOrganisation.Should().Be(organisationName);
+            Breadcrumb.ExternalActivity.Should().Be(BreadCrumbConstant.SchemeManageEvidence);
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNoteGet_GivenANewModelIsCreated_ModelWithDefaultValuesIsCreated()
+        {
+            // act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(OrganisationId, EvidenceNoteId) as ViewResult;
+            var model = result.Model as ReviewEvidenceNoteViewModel;
+
+            // assert
+            model.ViewEvidenceNoteViewModel.Should().Be(model.ViewEvidenceNoteViewModel);
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNoteGet_GivenANewModelIsCreated_ModelWithPossibleValuesIsCreated()
+        {
+            // act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(OrganisationId, EvidenceNoteId) as ViewResult;
+            var model = result.Model as ReviewEvidenceNoteViewModel;
+            var expected = new List<string> { "Approve" };
+
+            // assert
+            Assert.Equal<IList<string>>(expected, model.PossibleValues);
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNoteGet_GivenANewModelIsCreated_ModelWithEmptySelectedValueIsCreated()
+        {
+            // act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(OrganisationId, EvidenceNoteId) as ViewResult;
+            var model = result.Model as ReviewEvidenceNoteViewModel;
+
+            // assert
+            model.SelectedValue.Should().Be(string.Empty);
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNoteGet_GivenApprovedSelectedValueIsSet_ModelWithSelectedEnumValueIsCreated()
+        {
+            // act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(OrganisationId, EvidenceNoteId) as ViewResult;
+            var model = result.Model as ReviewEvidenceNoteViewModel;
+            //model.SelectedValue = "Approved";
```

Comment:
You will need to merge from develop as the text for the button has been updated per content